cypress/e2e/supervision/deputies/add-deputy.cy.js

```diff
@@ -6,6 +6,8 @@ beforeEach(() => {
 
   cy.get("@client").then(({ id }) => {
     cy.visit(`/supervision/#/clients/${id}`);
+    cy.get('.TABS_DOCUMENTS').click();
+    cy.waitForStableDOM();
```

Comment:
Is there anything you can replace this wait for stable DOM with? I think it's only usually a last resort and we try to wait for something we expect to see on the page (in this case I guess a document or title?)